docs/sample-solutions/fasttrack/hpc/hybrid-hpc.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Hybrid HPC in Azure
+description: Running hybrid HPC workloads in Azure and on-premises
+author: Mike Warrington and Ben Hummerstone
+ms.date: <publish or update date>
+---
+# Hybrid HPC in Azure
+
+High Performance Computing on Azure is somtimes also referred to as Big Compute, as it moves away from the traditional need for on-premises specialist hardware as a major investment. On-demand compute resources can be utilised when a buiness needs it the most, either with hardware for specific HPC scenarios (GPU or RDMA for instance) or general purpose VMs that provide a balance on cost and performance. 
+
+This sample solution demonstrates how CycleCloud* can be used to orchestrate an IaaS HPC grid running on Azure, while also providing an optional scenario to extend a grid running on-premises to Azure for on demand compute resources.
+
+By using CycleCloud, the manual work to create an IaaS grid in the cloud becomes an efficient process, and can also be used to faciliate a new grid for Disaster Recovery or failover purposes. Additionally, organisations can build multiple grids and be confident there is no infrastructure drift taking place between deployments.
+
+\* CycleCloud has been a recent aquisition for Microsoft, we are currently in the process of intregrating this solution with Azure. During this period the functionality of CycleCloud will not be modified.
+
+## Potential use cases
+
+You should consider this solution for the following use cases:
+
+* If you have been manually building HPC IaaS infrastructures or using an ARM template, but want to further automate the deployment process.
```

Comment:
This is the first time the term "ARM" has been used, since it's also a processor and this HPC document, we should define it here.

Reply:
Also please note, we don't use this acronym. Use "Azure Resource Manager" on first mention, then "Resource Manager" - see https://worldready.cloudapp.net/Styleguide/Read?id=2696&topicid=39369